client/my-sites/themes/helpers.js

```diff
@@ -104,6 +103,11 @@ export function getExternalThemesUrl( site ) {
 }
 
 export function trackClick( componentName, eventName, verb = 'click' ) {
+	// The analytics module is highly reliant on BOM and DOM, and while we can mock
+	// the entire module trivially on server build, we can't do the same for test
+	// builds, so importing it at the top level broke tests. This is why we're using
```

Comment:
Could we use https://github.com/Automattic/wp-calypso/blob/fcf789ebb43b7d8fe9a912d8fa82d5bee66573b5/client/state/analytics/test/helpers/analytics-mock.js instead?